lennykamande/andela_bootcamp_learning
# Bg coverall implement fix five 2110701

**What does this PR do?**
This is a shopping Application that allows one to create a list and add items to it

**How does one test it?**

1. Clone/ download the application
2. install python
3. install a virtual Environment 
4. run the command pip install -r requirements.txt
5. run the command "export FLASK_CONFIG=development"
6. run the command "export FLASK_APP=run.py
7. Run the command "flask run"
8. You should get a link to which you will follow to your browser

Create a user and add a password
After login
